src/components/common/YourCompany/YourCompany.vue

```diff
@@ -517,10 +510,6 @@ export default class YourCompany extends Mixins(
     // reset name request
     if (this.getOriginalIA) {
       this.setNameRequest(this.getOriginalIA.nameRequest)
-    } else if (this.getOriginalChgRegistration) {
-      this.setNameRequest(this.getOriginalChgRegistration.nameRequest)
-    } else if (this.getOriginalRegistration) {
-      this.setNameRequest(this.getOriginalRegistration.nameRequest)
```

Comment:
Firm corrections don't load the original Registration / ChangeOfRegistration. Instead, firm corrections load a business snapshot. So, this original else case works just fine.